new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Namespace.java

```diff
@@ -1,10 +1,12 @@
-// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Copyright 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Why did this get changed? The (c) is part of the check-style.

Reply:
Ah, Thanks.  this is a copy over issue.  I created a different branch for the PR and pulled over my modified file that has an older copyright. Thanks.

Reply:
Corrected copyright.